SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialClassInfoManager.h

```diff
@@ -39,6 +39,8 @@ FORCEINLINE ESchemaComponentType GetGroupFromCondition(ELifetimeCondition Condit
 		return SCHEMA_InitialOnly;
 	case COND_ServerOnly:
 		return SCHEMA_ServerOnly;
+	case COND_AuthServerOnly:
```

Comment:
not sure why it is complaining about this case in the include, even tried manually including the header its from and made no difference?